bedrock/base/templates/macros.html

```diff
@@ -53,8 +53,8 @@
 
 
 {# Docs: https://bedrock.readthedocs.io/en/latest/send-to-device.html #}
-{% macro send_to_device(platform='all', product='firefox', message_set='default', id='send-to-device', class='horizontal', include_title=True, title_text=None, input_label=None, legal_note_email=None, legal_note_sms=None, spinner_color='#000', input_label_alt=None, button_class=None) %}
-  <section id="{{ id }}" class="send-to-device {{ class }}" data-countries="{{ send_to_device_sms_countries(message_set) }}" data-spinner-color="{{ spinner_color }}">
```

Comment:
Is removing `data-spinner-color="{{ spinner_color }}` intentional here? Looks like it's still used by the JS? https://github.com/mozilla/bedrock/blob/master/media/js/base/send-to-device.js#L33